image/getting_started/configuration.md

```diff
@@ -96,3 +96,50 @@ Read more in the official PHP documentation for:
 * [upload_max_filesize](http://www.php.net/manual/en/ini.core.php#ini.upload-max-filesize)
 
 It's possible to set these directives in your [php.ini](http://www.php.net/manual/en/ini.core.php) or at runtime with [ini_set](http://www.php.net/manual/en/function.ini-set.php).
+
+---
+
+## Extending
+
+Since version 2.3.10 you're able to extend Intervention Image and let it to use your own driver for all its operations. Your driver has to extend the AbstractDriver. More likely you just want to change some behaviour in the default GD and Imagick drivers. To let Intervention Image use the driver, just pass the driver into configuration array, as shown in the example below.
+
+#### Example
+
+```php
+// include composer autoload
+require 'vendor/autoload.php';
```

Comment:
This line [magically guesses the path](http://php.net/manual/en/function.include.php) (starting with the include paths, not the current dir…) and isn't advisable.

Instead you should use `'./vendor'` (CWD) or `__DIR__.'/vendor'` (dir of the file).